app/models/meal.rb

```diff
@@ -18,6 +18,18 @@ def has_gut_feeling?
     gut_feeling ? true : false # ternary operator
   end
 
+  def localtime
+    created_at.localtime.to_datetime.strftime("%a, %b %d at %I:%M%P")
```

Comment:
I really like the way this format looks in the table, very user friendly and readable